cluster/addons/cluster-monitoring/stackdriver/heapster-controller.yaml

```diff
@@ -69,7 +71,8 @@ spec:
           command:
             - /monitor
             - --source=heapster:http://localhost:8082?whitelisted=stackdriver_requests_count,stackdriver_timeseries_count
-            - --stackdriver-prefix=container.googleapis.com/internal/addons
+            - --stackdriver-prefix={{ prometheus_to_sd_prefix }}/internal/addons
```

Comment:
That may be not a very good idea. It may happen that some people already configured alerts based on this name.